packages/components/src/Placeholder/shimmer/index.native.js

```diff
@@ -0,0 +1,22 @@
+// @TODO see if it is still usable somehow: react-native-shimmer must be linked, so it will not work in Expo.
```

Comment:
check [this](https://github.com/tomzaku/react-native-shimmer-placeholder#for-who-using-expo) it is usable with expo

Reply:
🙈 seems that is another package

Reply:
Thanks for suggestion!